tapdb/sqlc/queries/universe.sql

```diff
@@ -159,14 +159,49 @@ INSERT INTO universe_events (
 );
 
 -- name: QueryUniverseStats :one
-WITH num_assets As (
-    SELECT COUNT(*) AS num_assets
+WITH stats AS (
+    SELECT total_asset_syncs, total_asset_proofs
+    FROM universe_stats
+), num_groups AS (
```

Comment:
Minor naming nit: this is just the set of groups, and not the _number_ of them yet.